custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc.py

```diff
@@ -85,15 +86,20 @@ def aggregation_query(self):
                 THEN 1 ELSE 0 END,
             CASE WHEN
                 (count(*) filter (WHERE date_trunc('MONTH', date_cbe_organise) = %(start_date)s))>0
-                THEN 1 ELSE 0 END
+                THEN 1 ELSE 0 END,
+            icds_dashboard_thr_v2.image_count
             FROM "{ucr_table}" awc_location
             LEFT JOIN "{cbe_table}" cbe_table on  awc_location.doc_id = cbe_table.awc_id
             LEFT JOIN "{vhnd_table}" vhnd_table on awc_location.doc_id = vhnd_table.awc_id
+            LEFT JOIN icds_dashboard_thr_v2 on (awc_location.doc_id = icds_dashboard_thr_v2.awc_id AND
```

Comment:
This tablename should also be parameterized

Reply:
Hey, just wanted to clear my understanding. Since all other tables had to parameterised as their names are calculated through a function and this one is static therefore I put it like this. Also, it is used only one place in the query.
Is putting out as parameter is more of code convention to keep 'table name as parameter' everywhere same?

Reply:
I think it makes sense as a code convention, but also trying to understand where this table is being used. If you want to find all places this table is being used, you would need to search for both the tablename and the constnat. If you change this using the constant, then you only need to search for the constant

Reply:
Done!. passed as parameter